scripts/db.js

```diff
@@ -1,7 +1,8 @@
 const mysql = require('mysql');
 const util = require('util');
+import { MYSQL_CONFIG } from '@lib/constants';
 
-const connection = mysql.createConnection(process.env.CLEARDB_DATABASE_URL);
+const connection = mysql.createConnection(MYSQL_CONFIG.host ? MYSQL_CONFIG : process.env.CLEARDB_DATABASE_URL);
```

Comment:
same here - let's check if `CLEARDB_DATABASE_URL` since that should take precedence